apps/condo/domains/ticket/components/IncidentHints.tsx

```diff
@@ -196,7 +202,7 @@ export const IncidentHints: React.FC<IncidentHintsProps> = (props) => {
         )
         const notActualLastIncidents = await fetchIncidents(
             [SortIncidentsBy.WorkFinishDesc, SortIncidentsBy.CreatedAtDesc],
-            incidentIds, organizationId, IncidentStatusType.NotActual, 3
+            incidentIds, organizationId, IncidentStatusType.NotActual, 7
         )
```

Comment:
Maybe better to use one parameter - object.
For better readability

Reply:
fixed it